src/predis/mock.clj

```diff
@@ -375,6 +375,17 @@
       (swap! store assoc k vs')
       "OK"))
 
+  (core/ltrim [this k start stop]
+    (if > (start stop)
+      []
+    (let [vs (core/get this k)
+          index-range (util.range/indices-for vs start stop)
+          start' (take 1 index-range)
+          stop' (take-last 1 index-range)
+          vs' (subvec vs start' (inc stop'))
+          ]
```

Comment:
- Your if clause is missing a form for `>`. Also whitespace: clauses should align in `if`, i.e.

```clj
(if (condition)
  (then-form)
  (else-form))
```

- Would rename `index-range` to `indices`
- You can define `start'` and `stop'` with `first` and `last` respectively
- Closing brace of `let` on same line as last binding `(inc stop'))]`